### PR TITLE
docs: Additional lightweight styling docs

### PR DESCRIPTION
**Pull request by kazo0**
https://github.com/unoplatform/private/issues/355

### CODE REVIEW

File: doc/lightweight-styling.md [hunk 1]
```diff
@@ -34,6 +43,7 @@ All interactive controls have multiple states, such as **PointerOver** (mouse is
 	</Button.Resources>
 </Button>
 
+<!-- #3 -->
```
Comment (jeromelaban) [inline doc/lightweight-styling.md:46]: @kazo0 should we also add an example to explain how to override resources using themes?

**Review by jeromelaban — APPROVED** [review]

File: doc/lightweight-styling.md (hunk 1 above)
Comment (jeromelaban) [reply doc/lightweight-styling.md:46]: Yes, this will likely be a common scenario.

Comment (agneszitte) [reply doc/lightweight-styling.md:46]: An example using Alias like here would be good also I think
similar to https://learn.microsoft.com/fr-fr/windows/apps/design/style/xaml-styles#aliasing-control-resources-to-new-names

File: doc/lightweight-styling.md [hunk 2]
```diff
@@ -3,13 +3,16 @@ uid: uno.themes.lightweightstyling
...
+An example of the anatomy of the lightweight styling resources available for something like Uno Material’s FilledButtonStyle can be seen below. The provided XAML code and image depict a Button using the default FilledButtonStyle followed by a second button, also with FilledButtonStyle applied, but now with specific resource keys overridden to customize its appearance.
 
 ```xml
-<Button Content="Default Button Style" Style="{StaticResource MaterialFilledButtonStyle}" />
+<Button Content="Default Button Style" Style="{StaticResource FilledButtonStyle}" />
 
-<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+<Button Content="Overridden Button Style" Style="{StaticResource FilledButtonStyle}">
 	<Button.Resources>
+		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
```
[the remark this answers is not in the document]
Comment (kazo0) [reply doc/lightweight-styling.md:15]: true, ill add it back as a property on the button itself

**Comment by agneszitte** [comment]
@Mergifyio backport legacy/2x release/stable/3.0

**Review by agneszitte — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**